CHANGELOG.rst

```diff
@@ -4,6 +4,122 @@
 Synapse Changelog
 *****************
 
+v2.103.0 - 2022-08-04
+=====================
+
+Features and Enhancements
+-------------------------
+- Updates to the ``it``, ``ou``, and ``risk`` models.
+  (`#2778 <https://github.com/vertexproject/synapse/pull/2778>`_)
+
+  ``it:prod:soft``
+    Add a ``techniques`` secondary property to record techniques employed by
+    the author of the software.
+
+  ``ou:campaign``
+    Add a ``techniques`` secondary property to record techniques employed by
+    the campaign.
+
+  ``ou:org``
+    Add a ``techniques`` secondary property to record techniques employed by
+    the org.
+
+  ``ou:technique``
+    Add a form to record specific techniques used to achieve a goal.
+
+  ``ou:technique:taxonomy``
+    Add a form to record analyst defined taxonomy of different techniques.
+
+  ``risk:attack``
+    Add a ``techniques`` secondary property to record techniques employed
+    during attack.
+    Deprecate the following secondary properties, in favor of using light
+    edges.
+
+      - ``target``
+      - ``target:host``
+      - ``target:org``
+      - ``target:person``
+      - ``target:place``
+      - ``used:email``
+      - ``used:file``
+      - ``used:host``
+      - ``used:server``
+      - ``used:software``
+      - ``used:url``
+      - ``used:vuln``
+      - ``via:email``
+      - ``via:ipv4``
+      - ``via:ipv6``
+      - ``via:phone``
+
+  ``risk:compromise``
+    Add a ``techniques`` secondary property to record techniques employed
+    during the compromise.
+
+  ``risk:threat``
+    Add a form to record a threat cluster or subgraph of threat activity to
+    a single group.
```

Comment:
the 'to a single group' reads kinda weird, seems like an 'attributable' is missing somewhere

Reply:
That was taken from the doc property for the form :thinking:

Reply:
the doc prop is slightly different: "A threat cluster or subgraph of threat activity attributable to one group."